db/seeds.rb

```diff
@@ -1,22 +1,57 @@
 require 'csv'
 
+PROD_USERS = [
```

Comment:
This probably shouldn't be in a seeds file, but instead in a Rake task or just done once on the server. This type of code makes it had to deploy the app / merge this PR.

Reply:
Oh I see. I'll let Alex know about this. He worked on getting our testing environment all set up so that we wouldn't have any issues. Thank you for letting our team know!

Reply:
Sorry for the late diff update. I am in Nairobi right now and have limited internet access. I removed changes in `seeds.rb` and now it is similar to the version of the file on the head of `saasbook/master`.

Reply:
No problem at all, thanks Alex!